routes/route.js

```
@@ -1,15 +1,46 @@
+const pdfGen = require('../utils/pdfGenerator');
+
+// Hardcoded secret (Security Risk)
+const SECRET_KEY = 'hardcodedSecret!';
+
+// Unused function (Duplicate/Unused Code)
+function unusedFunc() {
+  return 'I am not used';
+}
+
+// Duplicate function (Duplicate/Unused Code)
+function unusedFunc() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'unusedFunc'. Consider to delete it or rename it.

'unusedFunc' is defined here:


(lint/suspicious/noRedeclare)